### PR TITLE
First merge [GEAR-476]

### PR DESCRIPTION
**Pull request by dpark6060**

### CODE REVIEW

**Comment by naterichman** [comment]
Will review this tomorrow!

**Comment by naterichman** [comment]
[Is there ever a case where this wouldn't be a file?  What should happen in that case?](https://github.com/flywheel-apps/fwgear-wsi-to-dicom-converter/blob/c830e30c981604ab3dc596061d84e99f66deeced/run.py#L104)

**Comment by dpark6060** [comment]
> [Is there ever a case where this wouldn't be a file? What should happen in that case?](https://github.com/flywheel-apps/fwgear-wsi-to-dicom-converter/blob/c830e30c981604ab3dc596061d84e99f66deeced/run.py#L104)

A directory maybe?  Not sure.  Good question.  Might have something to do with the output of the command. I'll double check.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        libgdk-pixbuf2.0-dev \
+        libcairo2-dev \
+        libsqlite3-dev \
+        libglib2.0-dev \
+        libxml2-dev \
+        libopenjp2-7-dev \
+        libgtest-dev
+
+ENV FLYWHEEL="/flywheel/v0"
+COPY ["requirements.txt", "/opt/requirements.txt"]
```
Comment (naterichman) [inline Dockerfile:23]: Possibly /tmp?

File: tests/unit_tests/test_dummy.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+
+def test_dummy():
```
Comment (naterichman) [inline tests/unit_tests/test_dummy.py:2]: Meant to be here?

File: Dockerfile (hunk 1 above)
Comment (naterichman) [inline Dockerfile:23]: test

File: tests/test/test_main.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
+import sys
+import os
+sys.path.append('/Users/davidparker/Documents/Flywheel/SSE/MyWork/Gears/WSI/fwgear-wsi-to-dicom-converter')
```
Comment (naterichman) [inline tests/test/test_main.py:3]: Maybe use relative path so that someone else could easily run this test without modifying the file

Comment (dpark6060) [reply tests/test/test_main.py]: actually very important, thanks

**Pull request merged**